Topics/Teamwork.md

```diff
@@ -1 +1,7 @@
 ## Resources for Teamwork
+
```

Comment:
Possibly move and adapt this section over to Task_Management_Software.md to accompany your other contribution, or add a reference to it for a reader of this file to quickly access.